HRInvoiceApp/HRInvoiceApp/AddCompany.xaml.cs

```diff
@@ -78,7 +76,7 @@ void saveCompany(object sender, EventArgs e)
                 DisplayAlert("Alert", "Graag een provincie invullen.", "OK");
                 return;
             }
-            if (ZipCode.Text.Count() != 6)
+            if (ZipCode.Text.Contains(" ") && ZipCode.Text.Count() == 6)
```

Comment:
Uhh wat doe je hier precies? ATM laat hij alleen een message zien wanneeer: Er een spatie in zit EN wanneer het aantal 6 is. Lijkt me vreemd... Dit klopt niet vgsmij.